src/components/ApplicationTile/index.tsx

```diff
@@ -0,0 +1,62 @@
+import { useTranslation } from 'react-i18next';
+import classNames from 'classnames';
+import Text from '../Text';
+import styles from './styles.css';
+import getFormattedTimeDifference from '~/utils/getFormattedTimeDifference';
+
+interface Props {
+  companyName: string;
+  roleName: string;
+  status: 'sent' | 'inProgress' | 'rejected' | 'accepted' | 'hired' | 'archived';
+  salary: {
+    from: number;
+    to: number;
+  };
+  linkToOffer: string;
+  createdAt: Date;
+  className?: string;
+}
+
+const ApplicationTile = ({
+  companyName,
+  roleName,
+  status,
+  salary,
+  linkToOffer,
+  createdAt,
+  className,
+}: Props) => {
+  const { t } = useTranslation('applications');
+
+  return (
+    <div className={classNames(styles.root, className)}>
+      <aside className={classNames(styles.aside, styles[status])} />
+      <main className={styles.main}>
+        <div className={styles.row}>
+          <Text variant='medium'>{companyName}</Text>
+
+          <span className={styles.statusWrapper}>
+            <div className={classNames(styles.statusDot, styles[status])} />
+            <div>{t(`status.${status}`)}</div>
+          </span>
+        </div>
+        <div className={styles.row}>
+          <Text>{roleName}</Text>
+          <Text variant='large' className={styles.salary}>
```

Comment:
To be discussed, but I also recommend to place salary, link to offer and createdAt as separated components

Reply:
I moved Link to separate component since it will probably be used in the future. But I would leave salary and createdAt here, because it is just a text and I'm not sure if the same styling will be used in other part of the application. If it is used with the same styling, we will create separate component.